libraries/ui-library/src/components/six-sidebar-item/six-sidebar-item.scss

```diff
@@ -2,11 +2,19 @@
   display: block;
 }
 
+::slotted(a) {
```

Comment:
Changing the color on hover doesn't currently work with this solution. Also the click area of the `a` tag only covers the text. The following fixes the hover and improves the click area, but doesn't include the padding of the parent:

```
::slotted(a) {
  text-decoration: none;
  color: inherit;
  width: 100%;
  display: block;
}
```